packages/react-dom/src/server/ReactPartialRenderer.js

```diff
@@ -442,18 +445,36 @@ function resolve(child, context) {
   return {child, context};
 }
 
+type Frame = {
+  domNamespace: string,
+  children: any,
+  childIndex: number,
+  context: Object,
+  footer: string,
+  debugElementStack: Array<React.Element>,
+};
+
 class ReactDOMServerRenderer {
-  constructor(element, makeStaticMarkup) {
+  stack: Array<Frame>;
+  exhausted: boolean;
+  currentSelectValue: null;
```

Comment:
This is not a very helpful annotation. It's not always `null`. So by encoding it like this we actually make it harder to fix the Flow coverage because now it *looks* like fixing this is a Flow violation.